lib/beautiful/init.lua

```diff
@@ -117,14 +117,25 @@ function beautiful.init(config)
         if type(config) == 'string' then
             -- Expand the '~' $HOME shortcut
             config = config:gsub("^~/", homedir .. "/")
+            configdir = config:gsub("/[^/]+$", "")
```

Comment:
Start awesome with arguments `-c relative/path/to/some/rc.lua`. This code will break.

I like this idea, but right now I don't know how to make it work reliably. How about instead giving the relative path to the config to the theme, so it can use it where needed?